spec_1/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "the_guessing_game",
```

Comment:
do you make separate package.json's for each spec? what's the purpose of doing this?